bridged-tokens-wrapper/src/lib.rs

```diff
@@ -247,7 +247,6 @@ pub trait BridgedTokensWrapper:
         &self,
         requested_token: TokenIdentifier,
         to: EthAddress<Self::Api>,
-        opt_refunding_address: OptionalValue<ManagedAddress>,
```

Comment:
How will the user be able to specify the optional refunding address?

Reply:
the user will not specify anything, being an OptionalValue, the last argument can be ignored. not even '@' needed

Reply:
The user will be the caller.
Dragos modified `bridge-proxy` so it won't call `bridged-tokens-wrapper`. 
Therefore, this endpoint is used only by the user.